Upstream: no
Reason: assumes being installed in /usr/bin and uses sick hacks to locate it's data.

diff --git a/bin/git-cola b/bin/git-cola
index 91eab7ec..6dc945c3 100755
--- a/bin/git-cola
+++ b/bin/git-cola
@@ -33,10 +33,10 @@ def setup_environment():
     # Look for modules in the source or install trees
     if python2:
         cola_mod = join(prefix, str('cola'), str('__init__.py'))
-        install_lib = join(prefix, str('share'), str('git-cola'), str('lib'))
+        install_lib = join(sys.prefix, str('share'), str('git-cola'), str('lib'))
     else:
         cola_mod = join(prefix, 'cola', '__init__.py')
-        install_lib = join(prefix, 'share', 'git-cola', 'lib')
+        install_lib = join(sys.prefix, 'share', 'git-cola', 'lib')

     if os.path.exists(cola_mod):
         # Source tree
diff --git a/bin/git-dag b/bin/git-dag
index 1cc1c34a..f9f7fa79 100755
--- a/bin/git-dag
+++ b/bin/git-dag
@@ -33,11 +33,11 @@ def setup_environment():

     if python2:
         cola_mod = join(prefix, str('cola'), str('__init__.py'))
-        install_lib = join(prefix, str('share'), str('git-cola'), str('lib'))
+        install_lib = join(sys.prefix, str('share'), str('git-cola'), str('lib'))
     else:
         # Look for modules in the source or install trees
         cola_mod = os.path.join(prefix, 'cola', '__init__.py')
-        install_lib = os.path.join(prefix, 'share', 'git-cola', 'lib')
+        install_lib = os.path.join(sys.prefix, 'share', 'git-cola', 'lib')

     if os.path.exists(cola_mod):
         # Source tree
